src/ServicePulse.Host/app/modules/monitoring/views/endpoint_details.html

```diff
@@ -54,13 +54,7 @@ <h1>
                 <div class="col-sm-12 no-side-padding graph-values">
                     <div class="row queue-length-values tooltip-trigger">
```

Comment:
If you also remove the link inside the tootltip, then remove the class `tooltip-trigger` and use a standard tooltip attribute for the metric explanation.